solidity/ecdsa/test/WalletRegistry.WalletCreation.test.ts

```diff
@@ -3420,24 +3457,22 @@ describe("WalletRegistry - Wallet Creation", async () => {
                 await assertDkgResultCleanData(walletRegistry)
               })
 
-              // it("should reward the notifier with tokens from DKG rewards pool", async () => {
-              //   const currentDkgRewardsPoolBalance =
-              //     await walletRegistry.dkgRewardsPool()
-              //   expect(
-              //     initialDkgRewardsPoolBalance.sub(currentDkgRewardsPoolBalance)
-              //   ).to.be.equal(params.sortitionPoolUnlockingReward)
-
-              //   const currentNotifierBalance: BigNumber = await testToken.balanceOf(
-              //     await thirdParty.getAddress()
-              //   )
-              //   expect(
-              //     currentNotifierBalance.sub(initialNotifierBalance)
-              //   ).to.be.equal(params.sortitionPoolUnlockingReward)
-              // })
-
               it("should unlock the sortition pool", async () => {
                 await expect(await sortitionPool.isLocked()).to.be.false
               })
+
+              it("should refund ETH", async () => {
+                const postNotifyThirdPartyBalance = await provider.getBalance(
+                  thirdParty.address
+                )
+                const diff = postNotifyThirdPartyBalance.sub(
+                  initThirdPartyBalance
+                )
+                expect(diff).to.be.gt(0)
+                expect(diff).to.be.lt(
+                  ethers.utils.parseUnits("20000", "gwei") // 0,00002 ETH
```

Comment:
Tests report different refund amounts for `notifySeedTimeout` & `notifyDkgTimeout`. We can do the same precision but it would be `0,002 ETH` for both with the current const gas values and the gas price set to 200 gwei. I think it makes sense to apply different precisions to be closer to the actual refund.